nautobot/ipam/tests/test_api.py

```diff
@@ -102,6 +103,74 @@ def setUpTestData(cls):
         }
 
 
+class VRFDeviceAssignmentTest(APIViewTestCases.APIViewTestCase):
```

Comment:
Could you add api test cases for `VRFDeviceAssignmentSerializer` `device` & `vrf` / `vm` & `vrf`

Reply:
[cf2a0bb](https://github.com/nautobot/nautobot/pull/5282/commits/cf2a0bb0d4ff60b404699990e8f9b3c20ccd64e7)